### PR TITLE
Add translations for views (#325)

### PR DESCRIPTION
**Pull request by cpvalente**
Adds localised strings for 

* Add foundation for views translation

* WIP: Move hard-coded strings to translation map

* Add foundation for views translation

* WIP: Translations

* Use flat Javascript object for translation mappings

* Revert lock file back to original state

* Rename getString() to getLocalizedString()

### CODE REVIEW

**Comment by cpvalente** [comment]
@kev-ac 

Sorry for rushing into it, I was hoping to get this lined up before going on holidays 
I have implemented the remaining strings and created placeholders in the german string object. What do you think?

Is it just a matter of updating the german strings?

File: apps/client/src/features/viewers/backstage/Backstage.tsx
Comment (kev-ac) [inline apps/client/src/features/viewers/backstage/Backstage.tsx:1]: This file still references getString() not getLocalizedString()

**Comment by kev-ac** [comment]
> I have implemented the remaining strings and created placeholders in the german string object. What do you think?

Placeholders for the german string object wouldn't be necessary because the lookup function falls back to the default language (EN) if it does not find a matching string in the requested language.

That being said: I cannot find the placeholders you've created in this PR. Am I missing something?

File: apps/client/src/translation/languages/de.ts
Comment (kev-ac) [inline apps/client/src/translation/languages/de.ts:1]: I would suggest removing the german placeholders before merging it into v2. This way the current behavior keeps the same. I will push the translations once I get to it.

**Review by kev-ac — CHANGES_REQUESTED** [review]
LGTM. See comment below

File: apps/client/src/translation/languages/de.ts
Comment (cpvalente) [reply apps/client/src/translation/languages/de.ts:1]: perfect, I have made the change and will merge.
Thank you again

**Pull request merged**